dashboard/installation/dashboard-migration.md

```diff
@@ -14,6 +17,9 @@ Invictus V2 includes functionality to automatically deploy role assignments whic
 
 - "Monitoring Contributor" rights for the FlowHandlerJob function for the Invictus resource group.
 
+If you previously has version 4.17 installed, you might have a lot of role assignments at the Invictus resource group level.
```

Comment:
This only happens if one of the experimental beta versions was installed. So I dont think it should be a core step for migration. Maybe it can be mentioned in the common issues step instead.